textdistance/libraries.py

```diff
@@ -97,6 +97,12 @@ def get_function(self):
                 # object constructor - the distance metric method is
                 # called dist_abs() (whereas dist() gives a normalised distance)
                 obj = getattr(module, self.func_name)().dist_abs
+            if self.module_name == 'rapidfuzz.string_metric':
+                if self.func_name in {'jaro_similarity', 'jaro_winkler_similarity'}:
+                    func = getattr(module, self.func_name)
+                    obj = lambda *args: func(*args) / 100
```

Comment:
For similarities RapidFuzz uses a range 0.0-100.0 instead of 0.0-1.0, so divide by 100